src/templates/_includes/platform_selector.html

```diff
@@ -0,0 +1,16 @@
+<mkt-select class="compat-filter">
+  <mkt-selected><p>
+    {{ _('Show me apps for <mkt-selected-text>{selectedPlatform}</mkt-selected-text>')
```

Comment:
Previously, it was a label for a select, which caused problems since it wasn't l10n-friendly.